hledger-lib/Hledger/Data/Types.hs

```diff
@@ -192,7 +193,7 @@ instance NFData AmountPrice
 data AmountStyle = AmountStyle {
       ascommodityside   :: Side,                 -- ^ does the symbol appear on the left or the right ?
       ascommodityspaced :: Bool,                 -- ^ space between symbol and quantity ?
-      asprecision       :: !Int,                 -- ^ number of digits displayed after the decimal point
+      asprecision       :: !Word8,               -- ^ number of digits displayed after the decimal point
```

Comment:
Since this appears in a bunch of type signatures, what do you think about a NumDigits type synonym ? 
[Testing commenting from IDEA.]

Reply:
Changed later I see.